thirdparty/jira.go

```diff
@@ -17,6 +17,13 @@ import (
 	"github.com/pkg/errors"
 )
 
+type JiraRelatedTickets struct {
```

Comment:
Could you use the old name of "SearchReturnInfo" for this? I think that more accurately describes this data structure, since it includes more than just tickets